todoist.el

```diff
@@ -335,6 +335,21 @@ P is a prefix argument to select a project."
                             `(("project_id" . ,(todoist--project-id (todoist--select-project)))))))
   (todoist))
 
+
+(defun todoist-quick-new-task (content due p)
+  "Create a new task without loading the todoist-buffer.
+
+CONTENT is the content string.
+DUE is the human friendly due string and can be empty.
+P is a prefix argument to select a project."
+  (interactive "sTask content: \nsDue: \nP")
+  (todoist--query "POST" "/tasks"
+                  (append `(("content" . ,content) ("due_string" . ,due))
+                          (when p
+                            `(("project_id" . ,(todoist--project-id (todoist--select-project)))))))
```

Comment:
We should avoid duplication. If this is a copy paste of lines 328-335, those lines should be exacted into a "private" function and shared by both `todoist-new-task` and `todoist-quick-new-task`.